### DISCUSSION TITLE
We need a search feature for a repository

### DISCUSSION THREAD

**Discussion by knd6060** [community/community#8802] [Mobile]
I want a search feature to find and open a certain file in the repo. I have repos with hundreds of files and many directories. The feature is on the web version already.

**Comment by brianlovin** [comment]
Thanks for the note @knd6060, agreed! We don't have any work on the roadmap right now to improve code browsing, but we know this is a rough part of the app that could be improved a lot. I'll add your +1 to our tracking issues 😊

**Comment by elisealix22** [comment]
Hi @knd6060 , we recently published a new version of GitHub Mobile (v1.133 and above) that adds code search to repositories. Thanks for being patient with us and let us know if you have any other suggestions!
